lib/bounces.js

```diff
@@ -108,6 +110,26 @@ module.exports = function (log, error) {
           }
         }
 
+        // Log flow metrics if `flowId` and `flowBeginTime` specified in headers
+        const flowId = getHeaderValue('X-Flow-Id', message)
+        const flowBeginTime = getHeaderValue('X-Flow-Begin-Time', message)
+        const elapsedTime = currentTime - flowBeginTime
+
+        if (flowId && flowBeginTime && (elapsedTime > 0)) {
+          const eventName = 'email.' + templateName + '.bounced'
+
+          // Flow events have a specific event and structure that must be emitted.
+          // Ref: https://github.com/mozilla/fxa-auth-server/blob/master/lib/metrics/context.js#L122
```

Comment:
Just a note on this comment, at some point `lib/metrics/context.js` will change then this line number will be wrong. Maybe just mention the file and function name or something?